cli/tests/pcluster/config/test_update_policy.py

```diff
@@ -1932,3 +1932,68 @@ def test_compute_and_login_nodes_stop_policy(
     if not expected_update_allowed:
         assert_that(update_policy.fail_reason(change, patch)).is_equal_to(expected_fail_reason)
         assert_that(update_policy.action_needed(change, patch)).is_equal_to(expected_action_needed)
+
+
+@pytest.mark.parametrize(
+    "old_storage_value, new_storage_value, expected_condition, expected_fail_reason, expected_action_needed",
+    [
+        pytest.param(
+            "/home",
+            "dummy",
+            False,
+            "The /home directory cannot be changed during an update",
+            "Please revert any changes to the /home mount",
+        ),
+        pytest.param(
+            "home",
+            "dummy",
+            False,
+            "The /home directory cannot be changed during an update",
+            "Please revert any changes to the /home mount",
+        ),
+        pytest.param(
+            "dummy",
+            "home",
+            False,
+            "The /home directory cannot be changed during an update",
+            "Please revert any changes to the /home mount",
```

Comment:
Nit: when changing from dummy to /home I'd expect an error message saying that i need to revert any changes to the dummy mount, not to /home.